ironfish-rust-nodejs/src/multisig.rs

```diff
@@ -447,3 +451,64 @@ pub struct DkgRound2Packages {
     pub encrypted_secret_package: String,
     pub public_packages: Vec<DkgRound2PublicPackage>,
 }
+
+#[napi(object, namespace = "multisig")]
+pub fn dkg_round3(
+    secret: &ParticipantSecret,
+    round2_secret_package: String,
+    round1_public_packages: Vec<String>,
+    round2_public_packages: Vec<String>,
+) -> Result<DkgRound3Packages> {
+    let round2_secret_package = hex_to_vec_bytes(&round2_secret_package).map_err(to_napi_err)?;
+    let round1_public_packages =
+        round1_public_packages
+            .into_iter()
+            .try_fold(Vec::new(), |mut vec, pkg| {
+                let pkg = hex_to_vec_bytes(&pkg).map_err(to_napi_err)?;
+                let pkg =
+                    dkg::round1::PublicPackage::deserialize_from(&pkg[..]).map_err(to_napi_err)?;
+                vec.push(pkg);
+                Ok::<_, napi::Error>(vec)
+            })?;
```

Comment:
nit - this is what `try_deserialize_public_packages` does. should we inline this logic in both places?

Reply:
Forgot about that method; added a more generic `try_deserialize` that can be reused in all places

Reply:
I did not remove `try_deserialize_identities` because that's used in relatively many places and I think it's nice to keep it